firstfile.py

```diff
@@ -0,0 +1,34 @@
+#!1.Заменить символ “#” на символ “/” в строке 'www.my_site.com#about'.
+
+a = 'www.my_site.com#about'
+b = a.replace('#','/')
+print(b)
```

Comment:
You do not need extra variable `b`

a = 'www.my_site.com#about'.replace('#','/')
print(a)

Reply:
corrected